src/build-time-render/BuildTimeRender.ts

```diff
@@ -53,6 +54,8 @@ function genHash(content: string): string {
 		.substr(0, 20);
 }
 
+const ignoreAdditionalChunks = ['main.js', 'bootstrap.js', 'runtime.js'];
```

Comment:
`runtime.js` is here because the tests use it. It may be better to remove this and migrate the tests to use `bootstrap.js` instead.